### PR TITLE
Updated tips for 2.8.0

### PR DESCRIPTION
**Pull request by psiinon**
Any other suggestions?

### CODE REVIEW

File: addOns/tips/src/main/resources/org/zaproxy/zap/extension/tips/resources/Messages.properties [hunk 1]
```diff
@@ -15,8 +15,7 @@ A lot of ZAP functionality is context sensitive as best accessed this way.
 tips.tip.gen.004 = Use keyboard shortcuts to speed up your testing - you can define your own combinations via 'Options / Keyboard' which also generates printable shortcut cheatsheets.
 tips.tip.gen.005 = If you are getting too many false positives try changing the threshold for that scan rule to High.\n\
 But also report the problem to us via the ZAP Users Group or Issues so we can investigate it - both of which are linked off the 'Online' menu
-tips.tip.gen.006 = Still using Filters? These are being phased out and will be removed in the future.\n\
-Use scripts instead - they are much more powerful.
+tips.tip.gen.006 = Filters are no longer supported. Use scripts instead - they are much more powerful.
```
Comment (thc202) [inline addOns/tips/src/main/resources/org/zaproxy/zap/extension/tips/resources/Messages.properties]: Mention also Replacer add-on?

File: addOns/tips/src/main/resources/org/zaproxy/zap/extension/tips/resources/Messages.properties [hunk 2]
```diff
@@ -34,16 +33,26 @@ Many of the ZAP core developers hang out on irc://irc.mozilla.org #websectools
...
 You can also 'pin' any tab you like so that it stays visible even after a restart.
 tips.tip.gen.017 = You can reorder table columns by dragging and dropping them.\n\
 You can also select which columns are show via the icon just above the scroll bar on the right hand side of each table.
-tips.tip.gen.018 =  Manual browse indicator - Pages found by the spiders and forced browser are flagged in the Sites tree with the relevant icon. This icon disappears when you visit that page manually. 
+tips.tip.gen.018 = Manual browse indicator - Pages found by the spiders and forced browser are flagged in the Sites tree with the relevant icon. This icon disappears when you visit that page manually.
+tips.tip.gen.019 = You can launch browsers that automatically proxy through ZAP and do not need to have the ZAP certificate added via the 'Manual Explore' button.
+
+# HUD tips
+tips.tip.hud.001 = The Heads Up Display brings all of the key ZAP functionality into your browser.
+tips.tip.hud.002 = Complete the the Heads Up Display tutorial - it covers all of the HUD's features.
```
Comment (thc202) [inline addOns/tips/src/main/resources/org/zaproxy/zap/extension/tips/resources/Messages.properties]: `the`

**Comment by psiinon** [comment]
Updated :)

**Comment by psiinon** [comment]
Added another HUD tip suggested by David :)

**Pull request merged**